package/src/custom/tests/School/unit-php/modules/Contacts/Student_GradebookTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+use Sugarcrm\SugarcrmTestsUnit\TestMockHelper;
+use Sugarcrm\Sugarcrm\Util\Uuid;
+
+require_once 'include/SugarQueue/SugarJobQueue.php';
+
+/**
+ * @coversDefaultClass \Students_Gradebook
+ */
+class Student_GradebookTest extends \PHPUnit_Framework_TestCase
```

Comment:
Another case where we switched to `\PHPUnit\Framework\TestCase` recently and you might need to update.